### PR TITLE
add schema for fetch-artifacts-pnc

### PR DESCRIPTION
**Pull request by hjmodi**
*  MMENG-1272

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/schemas/fetch-artifacts-pnc.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Artifacts to be fetched from PNC builds",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "builds": {
+      "type": "array",
+      "properties": {
```
Comment (twaugh) [inline atomic_reactor/schemas/fetch-artifacts-pnc.json:11]: I don't think an array can have properties. Perhaps make a separate `metadata` object alongside the `builds` array?

Comment (hjmodi) [reply atomic_reactor/schemas/fetch-artifacts-pnc.json:11]: Updated.

**Comment by MartinBasti** [comment]
Looks good, except missing parts :)

**Comment by hjmodi** [comment]
Closing this will rework and create new PR.

**Pull request closed without merging**